monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/web-app/src/home/Dashboard.js

```diff
@@ -81,10 +82,18 @@ const styles = (theme) => ({
 function Layout(props) {
     const { classes } = props;
     const [mobileOpen, setMobileOpen] = React.useState(false);
+    const { signIn } = useAuthContext();
 
     const handleDrawerToggle = () => {
         setMobileOpen(!mobileOpen);
     };
+
+    React.useEffect(() => {
```

Comment:
Shall we use `useEffect` directly instead of `React.useEffect` to maintain consistency ?

Reply:
Fixed with e90503f